basin3d/plugins/usgs.py

```diff
@@ -92,7 +93,7 @@ def map_statistic_code(stat_cd):
     for k, v in USGS_STATISTIC_MAP.items():
         if stat_cd == v:
             return k
-    return 'NOT_SUPPORTED'  # consider making this part of the Mixin Statistic type
+    return 'NOT SUPPORTED'  # consider making this part of the Mixin Statistic type
```

Comment:
I don't think this should be changed as it is a part of the colum name. @dschristianson can you confirm?
```suggestion
    return 'NOT_SUPPORTED'  # consider making this part of the Mixin Statistic type
```

Reply:
Also, this should not be tacked onto the column name when statistics are not supported.

Reply:
Agree. And the code change I believe should happen in the `get_timeseries_data` method where the column names are generated.

Reply:
Hi @heycatwonton:
There are some other spots to address wrt statistics -- looks like the enums are not hooked up. Let's make this into another issue ticket. 

WRT Val's comments: Your PR had a code change in new line 96 -- an underscore was removed. Val was saying that that line should not be changed. Also Val addressed not adding "NOT_SUPPORTED" to the column name in her recent PR. All-in-all, please don't change that line for this PR.